src/cloud-api-adaptor/docs/SecureComms

```diff
@@ -36,17 +36,41 @@ data:
     ...
 ```
 
-You may also include additional Inbounds and Outbounds configurations to the Adaptor using the `SECURE_COMMS_INBOUNDS` and `SECURE_COMMS_OUTBOUNDS` config points. 
+You may also include additional Inbounds and Outbounds configurations to the Adaptor using the `SECURE_COMMS_INBOUNDS` and `SECURE_COMMS_OUTBOUNDS` config points. See more details regarding Inbounds and Outbounds below.
+
 You may also set the KBS address using the `SECURE_COMMS_KBS_ADDR` config point.
 
+
+### Adding named tunnels to the SSH channel
```

Comment:
Also, this commit is doing more than introducing typos.

Reply:
Lets squash it :)